bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryClientFactoryConfig.java

```diff
@@ -20,6 +21,7 @@
   private final BigQueryProxyConfig bigQueryProxyConfig;
   private final Optional<String> endpoint;
   private final int cacheExpirationTimeInMinutes;
+  private final ImmutableMap<String, String> bigQueryJobLabels;
```

Comment:
Do we need to make any changes to this file? I ask because we bind `BigQueryConfig` to `SparkBigQueryConfig` here https://github.com/GoogleCloudDataproc/spark-bigquery-connector/blob/master/spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConnectorModule.java#L53

Reply:
The class implements BigQueryConfig, so it need to provide this method as well. It is used to carry the configuration to the executors to create the Read/Write clients. The interface is defined in the bigquery-connector-common and needs this method to provide it to the BigQueryClient creation